bayesian.go

```diff
@@ -253,6 +323,38 @@ func (c *Classifier) Learn(document []string, which Class) {
 	c.learned++
 }
 
+// Here we use all the TF samples for the class and convert
+// them to TD-IDF https://en.wikipedia.org/wiki/Tf%E2%80%93idf
+// once we have finished learning all the classes and have the totals
+func (c *Classifier) ConvertTermsFreqToTFIDF() {
+
+	if c.didConvertTfIdf {
+		panic("Error:TfIdf-Mode:Cumulative counts - can only call this once. Reset and relearn.")
```

Comment:
Change message to "Cannot call ConvertTermsFreqToTfIdf more than once. Reset and relearn to reconvert."